src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Power-Strings.json

```diff
@@ -182,6 +182,13 @@
       "You cannot remember other memories until you have #yClarity of your current memory."
     ]
   },
+  "stsjorbsmod:OldBookPower": {
+    "NAME": "Old Book",
+    "DESCRIPTIONS": [
+      "When you die, instead destroy Old Book.",
+      "When you die, instead destroy Old Book and heal #g%1d percent of your max HP."
```

Comment:
Lizard tail and Fairy in a bottle uses "When you **would** die" for the conditional, probably best to be consistent